README.md

```diff
@@ -242,8 +242,8 @@ Nodejs has two kinds of streams: [web streams](https://nodejs.org/dist/latest-v1
 
 #### Specification Compliance
 
-This section documents parts of the [Fetch Standard](https://fetch.spec.whatwg.org) which Undici does
-not support or does not fully implement.
+This section documents parts of the [Fetch Standard](https://fetch.spec.whatwg.org) that Undici does
+not support or fully implement.
```

Comment:
IMHO incorrect

Reply:
From rereading it the removal of the additional "does not" reduces clarity for the sake of conciseness, I agree.

Reply:
Updated in https://github.com/nodejs/undici/pull/1354/commits/4d9f60dd622662e6fc11b26a19d480cf17b4fc53